core/templates/dev/head/editor/ExplorationHistory.js

```diff
@@ -70,9 +107,18 @@ oppia.controller('ExplorationHistory', [
        */
       $scope.compareVersions = {};
       $scope.compareVersionMetadata = {};
+      
+      // Contains the IDs of the versions selected for comparison. Should contain a maximum of two elements.
+      $scope.selectedVersionsArray = []; 
 
       $scope.hideHistoryGraph = true;
-      $scope.hideCompareVersionsButton = false;
+
+      // Disable all comparisons if there are less than two revisions in total.
+      $scope.disableCompare = (currentVersion<2) ? true : false;
```

Comment:
nit: add spaces before and after the operator
This line can be simplified as
    $scope.disableCompare = (currentVersion < 2);